From 24f37387f6ed20dd29f2def3c4d1f64ba15d3b8f Mon Sep 17 00:00:00 2001
From: "Miss Islington (bot)"
 <31488909+miss-islington@users.noreply.github.com>
Date: Tue, 28 May 2019 20:12:30 -0700
Subject: [PATCH 01/28] bpo-26903: Limit ProcessPoolExecutor to 61 workers on
 Windows (GH-13132) (GH-13643)

Co-Authored-By: brianquinlan <brian@sweetapp.com>
(cherry picked from commit 39889864c09741909da4ec489459d0197ea8f1fc)

Co-authored-by: Brian Quinlan <brian@sweetapp.com>
---
 .../NEWS.d/next/Library/2019-05-06-19-17-04.bpo-26903.4payXb.rst | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 Misc/NEWS.d/next/Library/2019-05-06-19-17-04.bpo-26903.4payXb.rst

diff --git a/Misc/NEWS.d/next/Library/2019-05-06-19-17-04.bpo-26903.4payXb.rst b/Misc/NEWS.d/next/Library/2019-05-06-19-17-04.bpo-26903.4payXb.rst
new file mode 100644
index 0000000000..ec3aa05e90
--- /dev/null
+++ b/Misc/NEWS.d/next/Library/2019-05-06-19-17-04.bpo-26903.4payXb.rst
@@ -0,0 +1 @@
+Limit `max_workers` in `ProcessPoolExecutor` to 61 to work around a WaitForMultipleObjects limitation.
\ No newline at end of file
-- 
2.24.3 (Apple Git-128)

